docs/source/_gallery_original/index.rst

```diff
@@ -55,7 +55,7 @@ We welcome contributions from the community. If you would like to contribute, pl
 
 1. Fork the `SkyPilot repository <https://github.com/skypilot-org/skypilot>`__ on GitHub.
 2. Create a new folder for your own framework, LLM model, or tutorial under `llm/ <https://github.com/skypilot-org/skypilot/tree/master/llm>`__.
-3. Add a README.md, a SkyPilot YAML, and other necessary files to run the AI project.
+3. Add a README.md, a SkyPilot YAML, and other necessary files to run the AI project. (Relative paths are not supported. Please use GitHub URLs for the references to the files.)
```

Comment:
Was thinking this hint can exist  ~L16 as a comment for reviewers to pay attention to. To make this section shorter.